python/usb_console/run_simulation.py

```diff
@@ -117,7 +118,7 @@ def set_up_radios(self):
                 radio_data_name = radio_connected_device + "_radio"
                 radio_datastore = Datastore(radio_data_name, self.simulation_run_dir)
                 radio_logger = Logger(radio_data_name, self.simulation_run_dir)
-                radio_session = RadioSession(radio_connected_device, radio_datastore, radio_logger, self.radio_keys_config)
+                radio_session = RadioSession(radio_name, radio_datastore, radio_logger, self.radio_keys_config)
```

Comment:
This is to fix name clashes between the "FlightController" Teensy and the "FlightController" radio